04_sessions.html

```diff
@@ -109,89 +109,36 @@
-        <dd>The name of the desired browser as a string.</dd>
-        <dt><span id="capability-browserVersion">browserVersion</span></dt>
-        <dd>The version number of the browser, given as a string.</dd>
-        <dt><span id="capability-platformName">platformName</span></dt>
-        <dd>The OS that the browser is running on, matching any of the platform names given below.</dd>
-        <dt><span id="capability-platformVersion">platformVersion</span></dt>
-        <dd>The version of the OS that the browser is running on as a string.</dd>
-      </dl>
-
-      <section>
-        <h3>Platform Names</h3>
-        <p>The following platform names are defined and MUST be used, case sensitively, for the "platformName" unless the actual platform being used is not given in this list:</p>
-        <ul>
-          <li>android</li>
-          <li>ios</li>
-          <li>linux</li>
-          <li>mac</li>
-          <li>unix</li>
-          <li>windows</li>
-        </ul>
-        <p>In addition "any" MAY be used to indicate the underlying OS is either unknown or does not matter.</p>
-
-        <p>Implementors MAY add additional platform names. Until these are standardized, these MUST follow the conventions outlined in <a href="#extending-the-protocol">extending the protocol</a> section with the exception that the vendor prefix SHOULD omit the leading "-" character.<p>
-
-        <p class="note">For example, Mozilla's Firefox OS could be described as either "-MOZ-FIREFOXOS". The latter makes it easier for local ends to specify an enum of supported platforms and is therefore recommended in this case.</p>
-
-        <!-- TODO: Add section on comparing platforms -->
-
-      </section>
-    </section>
-    <section>
-      <h3>Error Handling</h3>
-      <p>The following status codes MUST be returned by the "newSession" command:</p>
-      <dl>
-        <dt><code><a href="#status-success">success</a></code></dt>
-        <dd>The session was successfully created. The "value" field of the <code>Response</code> MUST contain a <a>Capabilities</a> object describing the session.</dd>
-        <dt><code><a href="#status-timeout">timeout</a></code></dt>
-        <dd>The new session could not be created within the time allowed for command execution on the remote end. This
-        time MAY be infinite (in which case this status won't be seen). The "value" field of the Response SHOULD contain a string explaining that a timeout has
-        occurred, but it MAY be left as the null value or filled with the empty string.</dd>
-        <dt><code><a href="#status-unknown-error">unknown error</a></code></dt>
-        <dd>An unhandled error of some sort has occurred. The "value" field of the Response SHOULD contain a more detailed
-        description of the error as a string.</dd>
-      </dl>
+      <li><p>Let <var>session id</var> be the result of creating a new <a>session ID</a> and
+           appending to <a>list of active sessions</a>.</p></li>
```

Comment:
You need to define how to create a new session ID under Session, or we can do this later, but it should read something along the lines of:

```html
<li><p>Let <var>session id</var> be the result of <a>generating a UUID</a>.</p></li>
<li><p>Append <var>session id</var> to the <a>list of active sessions</a>.</p></li>
```